src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,57 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import javax.swing.text.Element;
+
+public class HomePage extends BasePage{
+    public HomePage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+    ///////////////////////////////////////////////////////////////////
+    /*By userAvatarIcon = By.cssSelector("img.avatar");
+
+    public WebElement getUserAvatar(){
+        return findElement(userAvatarIcon);
+    }
+    /////////////////////////////////////////////////////////////////////
+    /*private By selectedPlayList = By.xpath("//li[@class= 'playlist playlist']//a[contains(text(), 'My Play List')]");
+    private By selectedPLField = By.cssSelector("[name = 'name']");
+    private By renamedPlayList = By.xpath("//div[@class= 'success show']");
+
+    public void doubleClickSelectedPL(){
+        doubleClick(selectedPlayList);
+    }
+    public void cleanAndEnterNewNave(String newPlayListName){
+        findElement(selectedPLField).sendKeys(Keys.chord(Keys.COMMAND, "A", Keys.DELETE));
+        findElement(selectedPLField).sendKeys(newPlayListName);
+        findElement(selectedPLField).sendKeys(Keys.ENTER);
+    }
+    public String getRenamedNewPlayList(){
+        return findElement(renamedPlayList).getText();
+    }*/
+////////////////////////////////////////////////////////////////////////////////
+    ////////////////////////////////////////////////////////////////////////////
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.